.gitignore

```diff
@@ -8,7 +8,6 @@
 node_modules/
 npm-debug.log
 yarn-error.log
-**/yarn.lock
```

Comment:
had to add the `yarn.lock` files, otherwise yarn complains. I actually think it's good to have them in the repository

```
#!/bin/bash -eo pipefail
yarn
➤ YN0000: ┌ Project validation
➤ YN0057: │ purchasetester: Resolutions field will be ignored
➤ YN0057: │ react-native-purchases-ui: Resolutions field will be ignored
➤ YN0000: └ Completed
➤ YN0028: The lockfile would have been created by this install, which is explicitly forbidden.
➤ YN0000: Failed with errors in 0s 2ms

Exited with code exit status 1
CircleCI received exit code 1
```